src/main/java/io/xdb/core/persistence/schema_to_load/PersistenceSchemaToLoad.java

```diff
@@ -0,0 +1,74 @@
+package io.xdb.core.persistence.schema_to_load;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public class PersistenceSchemaToLoad {
```

Comment:
the names don't seem accurate to me Lol. 
It's not to load the schema, it's to load the global attributes. 
I guess you want to say "this is the schema to load the attributes"

Reply:
Renamed them to be `PersistenceSchemaToLoadData` and `PersistenceTableSchemaToLoadData`.

Reply:
I don't like the name(sorry). 

First of all the name is quite long, and second it doesn't convey the useful information. 

Our persistence schema is divided into two parts today: global attribute and local attribute. 
Where global attribute is mapped to database table. 

My suggestion to hierarchy naming:

PersistenceSchema  = GlobalAttributeSchema + LocalAttributeSchema + an optional list of of NamedPersistencePolicy

GlobalAttributeSchema = a list of DBTableSchema, each represents the schema mapped to a database table. 

DBTableSchema = PK names+types, column names + types, whether or not load by default, and the locking type. 

Optionally defined some namedPersistencePolicy: 
NamedPersistencePolicy = a string name + PersistencePolicy

PersistencePolicy = LocalAttributePolicy + GlobalAttributePolicy

GlobalAttributePolicy = a map of table name to the DBTablePolicy

DBTablePolicy = loading column names, locking type.

I am open to hear improvements, but I want to share with you why I come with these concepts first. You will find it easier to make improvements on what I have tried :D

Reply:
The name policies will be really useful for your implementation of RPC annotation. 

If you look at the iWF java sdk: https://github.com/indeedeng/iwf-java-sdk/blob/main/src/main/java/io/iworkflow/core/RPC.java#L20

There are a big list of annotation parameters. This is because Java annotation only allows primitive (string,int,bool) or a list of primitive types. Regular class/objects are not allowed. Using a string name will work like a "referrence" and avoid users from defining those over and over again (the parameters will be a lot especially using both global and local attributes)

Reply:
As discussed in Slack, policy will not be applied for now. Since the global attribtues have not been finalized, I will leave the codes as what they are for now. We can revisit them once persistence has been finalized.